src/main/java/com/blackducksoftware/tools/commonframework/core/config/ConfigurationProperties.java

```diff
@@ -80,7 +80,7 @@ public Properties getProperties() {
 		final Iterator<String> iter = config.getKeys();
 		while (iter.hasNext()) {
 			final String key = iter.next();
-			final String unEscapedValue = unescape(config.getString(key));
+			final String unEscapedValue = unescape(key, config.getString(key));
 			logger.debug("getProperties(): including: " + key + "=" + obscurePassword(key, config.getString(key) + " --> " + unEscapedValue));
 			propertiesObject.put(key, unEscapedValue);
 		}
```

Comment:
I think you also want to remove [ + " --> " + unEscapedValue] from line 84. (We lose the unescaped value for non-passwords, but I don't think that's really going to matter.)

Reply:
Wait why though? (sorry if this is trivial and it's going straight over my head). Passing in config.getString(key) + " --> " + unescapedValue will turn into ******** if the key is a password; It doesn't include the --> + unescapedValue part. So it doesn't log passwords and it still maintains the unescaped value for non-passwords.

Reply:
Sorry, my misunderstanding.
Once I get the build working, I'll ask you to merge those pom changes from master, and then we can merge your branch into master and work on releasing.